partitura/io/__init__.py

```diff
@@ -54,20 +54,15 @@ def load_score(filename: PathLike, force_note_ids="keep") -> Score:
     scr: :class:`partitura.score.Score`
         A score instance.
     """
-    part = None
-
-    # Catch exceptions
-    exception_dictionary = dict()
-    # Load MusicXML
-    try:
+    extension = filename.split(".")[-1].lower()
```

Comment:
I would consider spliting the extension using the dedicated os function to avoid error with strings that have multiple `.` in them. I propose:
```python
os.path.splitext(filename)[-1].lower()
````

Reply:
Subsequently, all if clauses should contain the dot, i.e. `".xml", ".mxl", ...`

Reply:
agree!